internal/start/start.go

```diff
@@ -418,7 +415,7 @@ EOF
 			"GOTRUE_SMS_OTP_EXP=6000",
 			"GOTRUE_SMS_OTP_LENGTH=6",
 			"GOTRUE_SMS_TEMPLATE=Your code is {{ .Code }}",
-			"GOTRUE_SMS_TEST_OTP=" + testOTP.String(),
+			fmt.Sprintf("GOTRUE_SMS_TEST_OTP=%s", testOTP.String()),
```

Comment:
Why do we need to change this line?

Reply:
@sweatybridge
Oh, my bad. I think deleted that while testing, and then re wrote it following the same pattern used for other envs.
Shall I change it and left it like before?

Reply:
@sweatybridge 
I have just went back to the previous implementation